CHANGES

```diff
@@ -49,6 +49,7 @@ Features added
 * HTML themes can set up default sidebars through ``theme.conf``
 * #3160: html: Use ``<kdb>`` to represent ``:kbd:`` role
 * #4212: autosummary: catch all exceptions when importing modules
+* #947: autodoc now supports ignore-dunder-all to ignore a module's ``__all__``
```

Comment:
What's the rational for the name? `dunder` has no meaning for me, at least

Reply:
It's a vaguely common-ish shortening (for certain values of "common") of "double underscore" to refer to \_\_variables\_\_ and \_\_methods\_\_.  (See [here](https://dbader.org/blog/python-dunder-methods), [here](https://stackoverflow.com/questions/33030434/dunder-methods-verus-builtin-methods), or [here](https://pythonconquerstheuniverse.wordpress.com/2012/03/09/pythons-magic-methods/) for a few random Google hits.)  I'd be happy to switch to a better name for the flag.  #947 suggests "hidden-members."  I don't really think of classes not in \_\_all\_\_ as "hidden," but I could switch to that if it's preferred.  Maybe "ignore-module-all" or "ignore-all-list"?

Reply:
I know the keyword, but I don't know it is common sense or not. So +1 for `ignore-module-all`.

Reply:
Thanks for that. I'd personally just squash and rebase, if that's ok by you.

Reply:
Sure, that's no problem at all.  Just re-pushed.